clients/ctl/admin-ui/src/types/errors/models.ts

```diff
@@ -30,3 +30,11 @@ export interface AlreadyExistsError {
     fides_key: string;
   };
 }
+
+export interface NotFoundError {
+  detail: {
+    error: string;
+    resource_type: string;
+    fides_key: string;
+  };
+}
```

Comment:
this is actually the same exact interface as `AlreadyExistsError`. I wasn't sure if I should just use the same definition, or keep things separate since they're technically different errors

Reply:
Maybe it makes sense to rename `AlreadyExistsError` to something like `ErrorDetails` or something and just re-use that?